package/test_requirements.txt

```diff
@@ -10,4 +10,3 @@ requests-mock>=1.6.0, <2.0
 flake8>=3.5, <4.0
 isort
 trufflehog>=2.0.99, <3.0
```

Comment:
Could you also update the lower bound of trufflehog to be 
`trufflehog>=2.1.0, <3.0` ?

Reply:
Thanks Kiyo! Some questions ...
(1) " .. update RELEASE.md with your name and Github link please? .."
I believe I should change RELEASE.md, then again: "git commit / git push / create a new PR, am I correct? 

(2) " ... add the description of which Github issue you are addressing in the PR ..."
Done! Please check ...

(3) " .. Could you also update the lower bound of trufflehog to be trufflehog>=2.1.0, <3.0 ? .."
Is this a new request, or it is inside the same Github issue?
Should I update RELEASE.md, update the lower bound of trufflehog and then "git commit / git push ..."?

Reply:
Sorry, one more question … Where exactly in the RELEASE.md file should I include my name and GitHub link please? Should I include a section in the beginning of the file such as:
## supporting contributions
[Ana Potje](https://github.com/ANA-POTJE)
Thanks!

Reply:
@ANA-POTJE 
You don't need to create a new PR. You can update RELEASE.md and lower bound of trufflehog (I suggested the change below, which you can accept), then "git commit/ git push" in this PR.
 
For the RELEASE.md, see how the previous contributors added their name in https://github.com/quantumblacklabs/kedro/blob/develop/RELEASE.md  

Let me know if anything is not clear :)

Reply:
For FYI, see how the person updated the RELEASE.md in a PR.  https://github.com/quantumblacklabs/kedro/pull/93/files